optimade/server/config.py

```diff
@@ -169,6 +169,12 @@ def load_from_ini(self):
         else:
             self.provider = self._DEFAULTS("provider")
 
+        self.aliases = {}
+        for endpoint in {"links", "references", "structures"}:
+            if f"{endpoint}.aliases" in config.sections():
```

Comment:
I don't think the explicit method call is needed.
```suggestion
            if f"{endpoint}.aliases" in config:
```